.github/workflows/ci_tests.yaml

```diff
@@ -59,15 +59,7 @@ jobs:
       # Install GMT and other required dependencies from conda-forge
       - name: Install GMT and required dependencies
         shell: bash -l {0}
-        run: |
```

Comment:
Notes: 

1. Remember to update the `ci_tests_dev.yaml` workflow.
2. Maybe take the opportunity to rename `ci_tests.yaml` to `ci-tests.yml`, and `ci_tests_dev.yaml` to `ci-tests-dev.yml` for consistency.

Reply:
>     1. Remember to update the `ci_tests_dev.yaml` workflow.

Not sure what's to update here since the dev workflow doesn't use `requirements[-dev].txt` at all?

>     2. Maybe take the opportunity to rename `ci_tests.yaml` to `ci-tests.yml`, and `ci_tests_dev.yaml` to `ci-tests-dev.yml` for consistency.

Prefer to keep this PR small-ish, and preserve the `git blame` :slightly_smiling_face:

>     3. Update the PR title
>     4. Update the PR description

Done and done!

Reply:
OK. Looks good.